docs/rdma/README.md

```diff
@@ -0,0 +1,7 @@
+# Running RDMA applications in Kubernetes
+
+RDMA supports zero-copy networking by enabling the network adapter to transfer data from the wire directly to application memory or from application memory directly to the wire, eliminating the need to copy data between application memory and the data buffers in the operating system. Such transfers require no work to be done by CPUs, caches, or context switches, and transfers continue in parallel with other system operations. This reduces latency in message transfer.
+
```

Comment:
apart from the high level introduction to RDMA , please explain (at high level) what is required for RDMA application to run in a container 

e.g.
 - RDMA capable hardware
 - RDMA modules loaded in the system
 - /dev/infiniband to be mounted to the container
 - IPC_LOCK capability

Reply:
+1,  list necessary pre-requisites and special config for rdma.